bazelrc/.bazelrc

```diff
@@ -1,50 +1,18 @@
-# Aspect-Recommended bazel flags for all projects
-# Docs for these flags: https://bazel.build/reference/command-line-reference
-# Accompanies this blog post: https://blog.aspect.dev/bazelrc-flags
+# Import Aspect recommended Bazel settings for all projects
+import %workspace%/.aspect/bazelrc/.bazelrc.common
 
-###
-# Performance fixes
-###
-# Consider --experimental_remote_merkle_tree_cache_size as well
-build --experimental_remote_merkle_tree_cache
-query --experimental_remote_merkle_tree_cache
-# Note: The noexperimental_check_external_repository_files flag requires Bazel 6
-build --noexperimental_check_output_files --noexperimental_check_external_repository_files
-fetch --noexperimental_check_output_files --noexperimental_check_external_repository_files
-query --noexperimental_check_output_files --noexperimental_check_external_repository_files
-build --incompatible_remote_results_ignore_disk
-# Observed to speed up an Angular build from 7.5min to 5min
-build --experimental_reuse_sandbox_directories
+# Import Aspect recommended Bazel 6 only settings for all projects
+import %workspace%/.aspect/bazelrc/.bazelrc.common6
 
-# Avoid waiting on 'Waiting for build events upload: Build Event Service'
-build --bes_upload_mode=fully_async
+# Import Aspect recommended Bazel settings for JavaScript projects
+import %workspace%/.aspect/bazelrc/.bazelrc.javascript
```

Comment:
I checked with the "official" vscode extension https://marketplace.visualstudio.com/items?itemName=BazelBuild.vscode-bazel&ssr=false#version-history

and `common.bazelrc.` gives a bazel icon while `.bazelrc.common` doesn't. So that's a hint maybe we should go with the former naming - do you have a source for which is more idiomatic/supported?

Reply:
I was going off our reasoning for the `.bazelrc.user` naming:
```
# Load any settings & overrides specific to the current user. `.bazelrc.user` should appear in
# `.gitignore` so that settings are not shared with team members. This needs to be last statement in
# this config, as the user configuration should be able to overwrite flags from this file. See
# https://bazel.build/configure/best-practices#bazelrc-file (Note that we use `.bazelrc.user` so the file
# appears next to `.bazelrc` in directory listing, rather than `user.bazelrc` as suggested in the
# Bazel docs)
try-import %workspace%/.bazelrc.user
```
but suppose if they are all in an `.aspect/bazelrc` folder the consideration is different there

Reply:
yeah maybe we should change that to `.aspect/bazel/user.bazelrc` as well, following this new pattern

Reply:
or maybe the `%workspace%/user.bazelrc` to follow Bazel's recommendation?

Reply:
meh, I don't think ppl should pollute their repo root with all our proposed *.bazelrc files, and the user one probably goes in the new location next to common.bazelrc

Reply:
sgtm

Reply:
✅